### PR TITLE
Feature - Remove lodash + Proper testing

### PR DESCRIPTION
**Pull request by marpme**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -31,11 +32,11 @@
...
+  "exclude": [
+    "node_modules",
     "tests/**/*.ts",
     "tests/**/*.tsx"
   ],
-  "exclude": [
-    "node_modules"
-  ]
-}
+}
```
Comment (SwenVanZanten) [inline tsconfig.json:42]: missing end of file space

File: .editorconfig [hunk 2]
```diff
@@ -1,5 +0,0 @@
-[*.{js,jsx,ts,tsx,vue}]
-indent_style = space
-indent_size = 2
-trim_trailing_whitespace = true
-insert_final_newline = true
```
Comment (SwenVanZanten) [inline .editorconfig:5]: why is this file removed?

File: tsconfig.json (hunk 1 above)
Comment (marpme) [reply tsconfig.json:42]: I don't know why this would be necessary? Can you maybe elaborate shortly?

Comment (SwenVanZanten) [reply tsconfig.json:42]: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Comment (SwenVanZanten) [reply tsconfig.json:42]: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

there's more where that came from 🤣

Comment (marpme) [reply tsconfig.json]: Thanks for getting back & solved :)
